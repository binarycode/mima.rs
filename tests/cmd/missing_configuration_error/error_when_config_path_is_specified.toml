args = 'list-guests'
status.code = 1
