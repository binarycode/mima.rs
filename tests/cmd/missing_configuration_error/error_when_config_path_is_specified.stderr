error: Missing configuration at './mima.toml', '/etc/mima.toml'
